### PR TITLE
Remove FTP server

### PR DESCRIPTION
**Pull request by valentinbreiz**
+ Make Tcp class and client StateMachine public for userland FTP implementation (coming soon as a nuget package)

### CODE REVIEW

File: IDE.sln [hunk 1]
```diff
@@ -44,22 +44,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Spruce", "..\XSharp\source\
...
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IL2CPU.Debug.Symbols.Net48", "..\IL2CPU\source\IL2CPU.Debug.Symbols.Net48\IL2CPU.Debug.Symbols.Net48.csproj", "{AC3FEEDE-DA07-40BF-A1EA-56716EBC2818}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
 		Release|x86 = Release|x86
+		TEST|Any CPU = TEST|Any CPU
```
Comment (KM198912) [inline IDE.sln:55]: Is there a reason for a new configuration?

Comment (valentinbreiz) [reply IDE.sln:55]: Visual Studio did it by itself

Comment (KM198912) [reply IDE.sln:55]: I don't think we need it as only debug is supported right now anyway

Comment (valentinbreiz) [reply IDE.sln:55]: What's the problem merging this?

**Comment by valentinbreiz** [comment]
https://www.nuget.org/packages/CosmosFtpServer/

File: IDE.sln (hunk 1 above)
Comment (KM198912) [reply IDE.sln:55]: Will the extra Configuration end up on the user's vs? If yes I think it's better to remove it before merging

Comment (quajak) [reply IDE.sln:55]: If think the Test configuration is needed for unit tests

**Review by quajak — APPROVED** [review]

**Pull request merged**